src/openscm_runner/adapters/fair_adapter/fair_adapter.py

```diff
@@ -38,36 +40,40 @@ def _make_full_cfgs(self, scenarios, cfgs):  # pylint: disable=R0201
             desc="Creating FaIR emissions inputs",
         ):
             smdf_in = ScmRun(smdf)
+            scen_startyear = smdf_in.time_points.years()[0]
             endyear = smdf_in.time_points.years()[-1]
-            emissions = scmdf_to_emissions(smdf_in, endyear=endyear)
+            emissions = scmdf_to_emissions(
```

Comment:
If you wanted to make the start year flexible, you could do something like

```
startyear = cfgs.pop("startyear", 1750)
scmdf_to_emissions(startyear=startyear)
```

That would let a user tell FaIR exactly which year to start in without having that config then propagate everywhere (I think)